modules/lib/src/main/resources/assets/js/app/settings/dialog/ProjectSelectionDialog.ts

```diff
@@ -137,4 +139,19 @@ export class ProjectSelectionDialog
         });
     }
 
+    // Cycle to the first project element when tabbing in the last element
+    private setLastProjectCycleOnTabEvent(): void {
+        if(this.projectsList.getItemViews().length === 0) return;
```

Comment:
According to our code conventions we always use brackets after `if` statement, no matter if it has only return after, so should be: 
```
if (this.projectsList.getItemViews().length === 0) {
            return;
}
```